jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/Constants.java

```diff
@@ -48,9 +48,9 @@ public class Constants {
 
     public static final String DIAGNOSTIC_CODE_REDUNDANT_DISPOSES = "RemoveExtraDisposes";
 
-    public static final Set<String> INVALID_INJECT_PARAMS = new HashSet<String>(Arrays.asList(DISPOSES, OBSERVES, OBSERVES_ASYNC));
-    public static final String[] INVALID_INJECT_PARAMS_FQ = { DISPOSES_FQ_NAME, OBSERVES_FQ_NAME,
-                                                              OBSERVES_ASYNC_FQ_NAME };
+    public static final Set<String> INVALID_CDI_PARAMS = new HashSet<String>(Arrays.asList(DISPOSES, OBSERVES, OBSERVES_ASYNC));
```

Comment:
I don't think reusing the variable is the way to go because it becomes confusing and hard to read. I think there should be the list of invalid parameters for initializer methods and producer methods, which are the things that are being handled here. For example: INVALID_INITIALIZER_METHOD_PARAMS, INVALID_PRODUCER_METHOD_PARAMS